app/views/layouts/searches_redesign.html.haml

```diff
@@ -0,0 +1,18 @@
+!!!strict
+%html
+  %head
+    %meta{ charset: 'UTF-8' }
+    %meta{ name: 'viewport', content: 'width=device-width, initial-scale=1.0' }
+    %meta{ 'http-equiv' => 'X-UA-Compatible', :content => 'IE=edge' }
+    = render_robots_meta_tag
+    = attribution
+    != "<!-- aid: #{@search.affiliate.id} handle: #{@search.affiliate.name} -->"
+    = favicon_link_tag favicon_url(@search.affiliate.favicon_url)
+
+    %title #{@page_title}
+
+    = javascript_pack_tag 'application'
+    = stylesheet_pack_tag 'application'
+
+  %body
+    #main-content{ tabindex: -1 }= yield
```

Comment:
Per our conversation, please remove any code that isn't explicitly required to satisfy the acceptance criteria for this story. Copying and pasting from the old HTML runs the risk of polluting our new SERP code with outdated/incorrect/untested code. For example, nothing currently tests the display of the favicon, the correct values of the robots meta tag, etc. For now, please pull out anything that's unnecessary and create follow-up stories to address needs (and discuss with @amanifarooque where those stories might land in the PRD). 

As for testing, the cucumber tests are good for testing what the user sees, but either view specs or system specs would be appropriate places to test the under-the-hood elements such as the robots meta tag.

Reply:
Thanks for pointing this out. I'll bring this up at tomorrow's grooming meeting.